submodules/LIB/hardware/simulation/Makefile

```diff
@@ -18,12 +18,15 @@ PYTHON_DIR = ../../scripts
 #include simulation headers and sources
 VHDR+=$(wildcard ./src/*.vh)
 VSRC+=$(wildcard ./src/*.v)
+VSRC+=$(wildcard ../common_src/*.v)
+
+VSRC+=$(wildcard $(REMOTE_SIM_DIR)/src/*.v)
```

Comment:
I don't understand why this line was added.
For the same reason as the comment in this line: https://github.com/IObundle/iob-soc/pull/817/files#diff-42165db73802a8e91c585e3325dd68d0e67ddd7255af5169486a6435f7f291beR39

Reply:
resolved. the line that i added was wrong, deleted and everything is working well.